### ISSUE TITLE
wi.relations is not a guaranteed property. Causing exceptions when not present

### ISSUE THREAD

**Issue by Danphillipz** [rfennell/AzurePipelines#2005]
### Azure DevOps Extensions

Generate Release Notes (Node Cross Platform)

### Platform

Azure DevOps Services

### Extension Version

4.20.2

### Describe the bug

I raised #2003 a couple of days ago, your change fixed that issue, but then the script failed on the next check regarding `wi.relations`.

See the logging information for the stacks trace on `getAllDirectRelatedTestCases` failing. 

Might be worth scanning through and checking that wherever we access the wi.relations property, we first check that it exists. 

### Repo Steps

In an azure board, I created a Spike user story with no relations.

I then added this to my github commit message using the AB#123 format.

Upon the release note generation step running in my pipeline it threw the error seen in the logs as there was no relations attribute present. While this did not entirely fail the generation, it did results in a partial success of the build

### Logging Information

```
TypeError: Cannot read properties of undefined (reading 'length') 
Total Builds: [0] 
at /home/vsts/work/_tasks/XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb/4.20.2/ReleaseNotesFunctions.js:791:60 
at Generator.next (<anonymous>) 
at /home/vsts/work/_tasks/XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb/4.20.2/ReleaseNotesFunctions.js:8:71 
at new Promise (<anonymous>) 
at __awaiter (/home/vsts/work/_tasks/XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb/4.20.2/ReleaseNotesFunctions.js:4:12) 
at getAllDirectRelatedTestCases (/home/vsts/work/_tasks/XplatGenerateReleaseNotes_7b0a5887-75c4-4ade-a915-f92a93c934cb/4.20.2/ReleaseNotesFunctions.js:786:12) 

```

**Comment by rfennell** [comment]
I need to add some more guards, I will get a new build out as soon as I can

**Comment by rfennell** [comment]
Found two places this could occur, will be fixed in 4.21.2

**Comment by Danphillipz** [comment]
Thanks @rfennell!
